src/components/DetailsMetrics/DetailsMetrics.scss

```diff
@@ -0,0 +1,189 @@
+@import '~igz-controls/scss/shadows';
```

Comment:
styles that related to the `MetricChart` component should be in the `MetricChart.scss` file and not here

Reply:
done.